.github/workflows/task-test.yml

```diff
@@ -202,26 +202,49 @@ jobs:
           echo "name=$(echo "${{ inputs.container || inputs.env }} ${{ runner.arch }}, Redis ${{ inputs.get-redis || 'unstable' }}" | \
                        sed -e 's/[":\/\\<>\|*?]/_/g' -e 's/__*/_/g' -e 's/^_//' -e 's/_$//')" >> $GITHUB_OUTPUT
       - name: Build
-        run: make SAN=${{ inputs.san }} REDIS_VER=${{ inputs.get-redis }}
+        env:
+          SAN: ${{ inputs.san }}
+          REDIS_VER: ${{ inputs.get-redis }}
+          DEBUG: 1
+        run: make
       - name: Unit tests
         timeout-minutes: ${{ fromJSON(inputs.test-timeout) }}
         id: unit_tests
         continue-on-error: true
-        run: make unit-tests LOG=1 CLEAR_LOGS=0 SAN=${{ inputs.san }}
+        env:
+          SAN: ${{ inputs.san }}
+          LOG: 1
+          CLEAR_LOGS: 0
+          DEBUG: 1
+        run: make unit-tests
       - name: Flow tests (standalone)
         timeout-minutes: ${{ fromJSON(inputs.test-timeout) }}
         id: standalone_tests
         if: ${{ inputs.standalone }}
         continue-on-error: true
-        run: make pytest SA=1 LOG=1 CLEAR_LOGS=0 SAN=${{ inputs.san }} ${{ inputs.test-config }}
-          REJSON=${{ env.REJSON }} REJSON_BRANCH=${{ inputs.rejson-branch }}
+        env:
+          SAN: ${{ inputs.san }}
+          LOG: 1
+          CLEAR_LOGS: 0
+          SA: 1
+          REJSON: ${{ inputs.rejson }}
+          REJSON_BRANCH: ${{ inputs.rejson-branch }}
+          DEBUG: 1
+        run: make pytest ${{ inputs.test-config }}
       - name: Flow tests (coordinator)
         timeout-minutes: ${{ fromJSON(inputs.test-timeout) }}
         id: coordinator_tests
         if: ${{ inputs.coordinator }}
         continue-on-error: true
-        run: make pytest SA=0 LOG=1 CLEAR_LOGS=0 SAN=${{ inputs.san }} ${{ inputs.test-config }}
-          REJSON=${{ env.REJSON }} REJSON_BRANCH=${{ inputs.rejson-branch }}
+        env:
+          SAN: ${{ inputs.san }}
+          LOG: 1
+          CLEAR_LOGS: 0
+          SA: 0
+          REJSON: ${{ inputs.rejson }}
+          REJSON_BRANCH: ${{ inputs.rejson-branch }}
+          DEBUG: 1
```

Comment:
Suggestion: Not real `DEBUG` but only to allow assertions. Consider something like `DIAG: 1` for diagnostics

Reply:
See added `ENABLE_ASSERT` compilation flag